### PR TITLE
[WIP] Add slideshow.php

### PR DESCRIPTION
**Pull request by andi34**
Original change here: https://github.com/fdschneider/photobooth/commit/39b148eb3a79e6bc3a5716f057d39708dd4da801

Had to do some adjustments to work on v2.x.

- [X] add back the shuffle function to show a random picture
  - [X] allow to disable shuffle function (instead show latest image first)
- [X] make refresh time adjustable via config/admin panel
- [X] make time a picture is shown adjustable via config/admin panel

### CODE REVIEW

**Comment by sualko** [comment]
What do you mean by "back shuffle function"?

Is there a reason why we don't use photoswipe for the slideshow? It offers a function [next()](https://photoswipe.com/documentation/api.html) which allows to create a slideshow quite easily.

**Comment by andi34** [comment]
Shuffle the array instead by date, so it shows a random picture

**Comment by sualko** [comment]
> which displays the photo # before the picture is taken

There are two approaches:

- If we assume that we have a random number generated with an equal distribution you could just pick a random image (e.g. `pictures[Math.round(Math.random()*pictures.length)]`)
- If we want to make sure that every picture is shown we have to create a new list and display one element after another (e.g. `picturesRandom =[]; while(pictures.length){i = Math.round(Math.random()*pictures.length); picturesRandom.push(pictures[i]); pictures = pictures.filter((el, index) => i !== index) //remove element from array}`)

**Comment by andi34** [comment]
fixed eslint errors

File: config/config.inc.php [hunk 1]
```diff
@@ -70,6 +70,9 @@
 $config['show_date'] = false; // only works if file_format_date = true
 $config['gallery']['date_format'] = 'd.m.Y - G:i';
 
+// SLIDESHOW
+$config['slideshow_refresh_time'] = '60';
```
Comment (andi34) [inline config/config.inc.php]: `refreshTime` to match the other config options for slideshow

**Comment by andi34** [comment]
In addition to https://github.com/andreknieriem/photobooth/pull/199 we'll also need https://github.com/andi34/photobooth/commit/3bbf4ae0d9919ab3012b6e1127d93a5b1dc20087

I am quite happy with the slideshow, maybe some code parts can be optimized.

**Comment by andi34** [comment]
Screenshot_20191124-104728

One Thing I don't know how to fix is how to adjust the layout in landscape. On my OnePlus 6 the image isn't completely visible.

Screenshot_20191124-104743

**Comment by sualko** [comment]
As I said it would be probably better to use photoswipe, because it has already solved all those issues.

**Pull request closed without merging**